internal/bzlmod/default_gazelle_overrides.bzl

```diff
@@ -22,60 +22,29 @@ DEFAULT_BUILD_FILE_GENERATION_BY_PATH = {
 }
 
 DEFAULT_DIRECTIVES_BY_PATH = {
-    "github.com/census-instrumentation/opencensus-proto": [
-        "gazelle:proto disable",
-    ],
     "github.com/envoyproxy/protoc-gen-validate": [
         "gazelle:build_file_name BUILD.bazel",
-    ],
-    "github.com/cockroachdb/errors": [
-        "gazelle:proto disable",
-    ],
-    "github.com/gogo/googleapis": [
-        "gazelle:proto disable",
-    ],
-    "github.com/gogo/protobuf": [
-        "gazelle:proto disable",
+        "gazelle:proto disable_global",
```

Comment:
Context for this line (and the others that re-introduce `gazelle:proto disable_global`)

given how overrides work, when an override is provided it will override the default behavior. This means existing overrides that wish to have the new default behavior, will need to now include `gazelle:proto disable_global` themselves. ()

This feels a little odd, so we should discuss. As far as I see we have the following options:

a. do what this PR does
b. don't bother introducing `gazelle:proto disable_global` to this DEFAULT_DIRECTIVES unless the mod has proto files already
c. invest in more advanced merging so that the defaults are not lost during merging. This would then also require the ability selectively override existing defaults

Reply:
I think we should do b: if those modules don't have `gazelle:proto disable` already, it either means they don't have proto files or people still need the proto files.